code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -666,12 +665,19 @@
 	required_container = /obj/item/slime_extract/rainbow
 
 /datum/chemical_reaction/slime/slimebomb/on_reaction(datum/reagents/holder, created_volume)
-	var/obj/item/grenade/clusterbuster/slime/volatile/S = new (holder.my_atom.loc)
+	var/obj/item/grenade/clusterbuster/slime/volatile/S = new (get_turf(holder.my_atom))
 	S.visible_message("<span class='danger'>Infused with slime jelly, the core begins to expand uncontrollably!</span>")
 	S.icon_state = "[S.base_state]_active"
 	S.active = TRUE
 	addtimer(CALLBACK(S, /obj/item/grenade.proc/prime), rand(15,60))
-	qdel(holder.my_atom) //deleto
+	var/turf/T = get_turf(holder.my_atom)
```

Comment:
if you put this above 668 you don't have to call it twice

Reply:
Alright, it is done.